backend/app/notify/email.go

```diff
@@ -19,11 +19,11 @@ import (
 
 // EmailParams contain settings for email notifications
 type EmailParams struct {
-	From                 string // from email address
-	MsgTemplate          string // request message template
-	VerificationSubject  string // verification message subject
-	VerificationTemplate string // verification message template
-	UnsubscribeURL       string // full unsubscribe handler URL
+	From                     string // from email address
+	MsgTemplatePath          string // request message template
+	VerificationSubject      string // verification message subject
+	VerificationTemplatePath string // verification message template
```

Comment:
I would suggest making `msgTemplatePath` and `verificationTemplatePath` non-public (by small first letter) because there is no way to change them from code anyway, and document path to templates somewhere in Readme.